sci-libs/pdal/pdal-2.4.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="A C++ library for translating and manipulating point cloud data"
+HOMEPAGE="https://pdal.io/"
+SRC_URI="https://github.com/PDAL/PDAL/releases/download/${PV}/PDAL-${PV}-src.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/13"
+KEYWORDS="~amd64 ~x86"
+IUSE="postgres test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+RDEPEND="
+	dev-libs/jsoncpp:=
+	net-misc/curl
+	sci-libs/gdal:=
+	sci-libs/libgeotiff:=
+	sci-geosciences/laszip
+	sys-libs/libunwind:=
+	sys-libs/zlib:=
+	postgres? ( dev-db/postgresql:*[xml] )
+"
+DEPEND="
+	test? ( sci-libs/gdal[geos,jpeg,png] )
+	${RDEPEND}
+"
+
+S="${WORKDIR}/PDAL-${PV}-src"
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_PLUGIN_PGPOINTCLOUD="$(usex postgres)"
+		-DWITH_LAZPERF=OFF
+		-DWITH_LASZIP=ON
```

Comment:
```
 * One or more CMake variables were not used by the project:
 *   WITH_LASZIP
 *   WITH_LAZPERF
```

Also leading to:

```
 * VDB: detected possibly incorrect RDEPEND (sci-libs/pdal-2.4.0)
 * dev-db/postgresql:14   < 
 * dev-libs/jsoncpp:=     < 
 *                        > app-arch/zstd:=
 *                        > dev-libs/libxml2
 *                        > dev-libs/openssl:=
 * sci-geosciences/laszip < 
```
(unneeded dep)

Reply:
f.yi.i it seems postgres is still needed:
- -- Found PostgreSQL: /usr/lib64/libpq.so (found version "13.5")  
-  * PostgreSQL PointCloud plugin, read/write PostgreSQL PointCloud objects